api/.travis.env

```diff
@@ -16,7 +20,7 @@ DB_USERNAME=postgres
 DB_PASSWORD=
 
 CACHE_DRIVER=redis
-# SESSION_DRIVER=memcached
+SESSION_DRIVER=array
```

Comment:
@artstorm whoa cool tool. Yea I think that is minimal impact though maybe worth creating an issue in github as a nice to have later.